proto/prysm/v1alpha1/beacon_block.proto

```diff
@@ -53,7 +53,6 @@ message GenericSignedBeaconBlock {
         SignedBlindedBeaconBlockDeneb blinded_deneb  = 8;
     }
     bool is_blinded = 100;
-    uint64 payload_value = 101;
```

Comment:
not used anywhere

Reply:
is it safe to remove here though instead of just deprecate ( if someone that's not us happens to use this )